Tests/SourceKitLSPTests/CodeLensTests.swift

```diff
@@ -0,0 +1,115 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LSPTestSupport
+import LanguageServerProtocol
+import SKTestSupport
+import SourceKitLSP
+import XCTest
+
+final class CodeLensTests: XCTestCase {
+  // MARK: - Helpers
+
+  func performCodeLensRequest(text: String, range: Range<Position>? = nil) async throws -> [CodeLens]? {
+    let testClient = try await TestSourceKitLSPClient()
+    let uri = DocumentURI(for: .swift)
+
+    testClient.openDocument(text, uri: uri)
+
+    let request = CodeLensRequest(textDocument: TextDocumentIdentifier(uri))
+    return try await testClient.send(request)
+  }
```

Comment:
This appears to be only used once, so I don’t think it’s worth having a helper function.